addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -21,6 +22,24 @@ weWidgets.LinkPopoverWidget.include({
 
         return this._super(...arguments);
     },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _onEditLinkClick(ev) {
+        this._super(...arguments);
+        if (isImg(this.$target[0].children[0])) {
+            const urlField = document.querySelector('[data-name="media_url_opt"] input');
+            if (urlField) {
+                urlField.focus();
+                urlField.select();
+            }
+        }
+    },
```

Comment:
AFAIK, the website module depends on the web_editor module but the reverse is not true. It is possible to have the web_editor module without website. Therefore, this piece of code will only work in website (not in email marketing for example). Can you change this? To be sure that it works I strongly advise you to test on the email marketing application, you will see that when you click on the edit button of the popover nothing happens for the moment

Reply:
After investigation, moving it to `web_editor` did not fix the problem: in mass mailing, the `document` is the root backend page document, while the editable content and the options pane are inside an iframe.
The selector was adapted to locate the correct document used by `wysiwyg`.